skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -76,6 +79,18 @@ abstract class AbstractSkaffoldRunConfiguration(
      * Persisted Skaffold config file absolute path for Skaffold run configurations.
      * See more at [com.intellij.openapi.vfs.VirtualFile.getPath]
      */
-    @Attribute("skaffoldConfigurationFilePath")
+    @get:OptionTag("skaffoldConfigurationFilePath")
```

Comment:
just for my own knowledge. what is this annotation?

Reply:
I think this is called Annotation Use-site Targets per Kotlin docs, they are using this for persisting Kotlin based properties. I think this limits this annotation to certain targets generated for annotation, for this case for getters, I saw this in `RunConfigurationOptions`.

Reply:
I actually meant the OptionTag part. The other info is good to know though!

Reply:
https://github.com/JetBrains/intellij-community/tree/master/platform/util/src/com/intellij/util/xmlb/annotations#readme

Seems like this is for backwards compatibility. But I guess we have no choice because we are stuck with the base class that is using this style?

Reply:
Hmm, you're right, apparently it just works without this tags at all, and in completely similar manner. Thanks for the hint, this makes it easier!

Reply:
thats strange. I wonder why it works without it? how does it know what to serialize now?

Reply:
sg